test-workflows/pipelines/generators/clusterloader/eks/tests/pipeline.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: loadtest
+spec:
+  workspaces:
+    - name: secrets
+    - name: config
+    - name: source
+    - name: results
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin.
+  - name: eks-version
+    default: "1.20"
+    description: The EKS version to install.
+  - name: region
+    description: The region where the cluster is in.
+    default: "us-west-2"
+  - name: zones
+    description: The zones where the cluster is in.
+    default: "us-west-2a,us-west-2b,us-west-2c"
+  - name: nodegroup-name
+    default: "linux-nodes"
+    description: The name of the nodegroup of the cluster.
+  - name: node-type
+    default: "m5.xlarge"
+    description: The type of the EC2 instaces for the nodegroup of the cluster.
+  - name: desired-nodes
+    default: "100"
+    description: The desired number of nodes in the cluster.
+  - name: min-nodes
+    default: "1"
+    description: The minimum number of nodes in the cluster.
+  - name: max-nodes
+    default: "100"
+    description: The maximum number of nodes in the cluster.
+  - name: giturl
+    description: "git url to clone the package"
+    default: https://github.com/kubernetes/perf-tests.git
+  - name: pods-per-node
+    description: "pod density"
+    default: "10"
+  - name: nodes-per-namespace
+    description:  "nodes per namespace to get created for load test "
+    default: "10"
+  - name: cl2-load-test-throughput
+    description: " throughput used for mutate operations"
+    default: "10"
+  tasks:
+  - name: create-eks-cluster
+    taskRef:
+      name: eks-cluster-create
+    params:
+      - name: cluster-name
+        value: '$(params.cluster-name)'
+      - name: eks-version
+        value: '$(params.eks-version)'
+      - name: region
+        value: '$(params.region)'
+      - name: zones
```

Comment:
Why can't we autogenerate zones based off of region. Can we just skip this one?

Reply:
https://eksctl.io/usage/creating-and-managing-clusters/
```
In us-east-1 you are likely to get UnsupportedAvailabilityZoneException. If you do, copy the suggested zones and pass --zones flag, e.g. eksctl create cluster --region=us-east-1 --zones=us-east-1a,us-east-1b,us-east-1d. This may occur in other regions, but less likely. You shouldn't need to use --zone flag otherwise.
```

Reply:
This is so weird. Have you seen this one yet?